db/1.9/modules/ROOT/pages/database-configure-data-types.adoc

```diff
@@ -0,0 +1,78 @@
+= Configure Database Connector Data Types Examples - Mule 4
+
+The following examples shows how to configure Anypoint Connector for Database (Database Connector) data types by using the Parameters Type field to specify the type of each input parameter in the query, and the Column Types field to set custom data types when connected to a particular database provider.
+
+== Configure the Parameter Types Field in Studio
+
+If your JDBC driver cannot determine the type of the input, or a parameter is of a custom type, configure the Parameter Types field to specify the type of each input parameter. The following example shows how to configure the Parameter Type field in the Bulk Insert operation:
+
+. In your Studio flow, select the *Bulk insert* operation.
+. In the *General* tab of the operation configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. Specify the database connection information and click *OK*.
+. Set the *SQL Query Text* field to the SQL query to execute, for example: `INSERT INTO PLANET(POSITION, NAME) VALUES (:position, :name)`
+. In the operation configuration screen, click the *Advanced* tab.
+. Set the *Parameter types* field to `Edit inline`.
+. Click the green plus sign (*+*) to create a new parameter type.
+. In the parameter type window, set the *Key* field to `name` and the *Type* field to `VARCHAR`.
+. Click *Finish*.
+. Repeat step 7.
+. In the parameter type window, set the *Key* field to `position` and the *Type* field to `INTEGER`.
+. Click *Finish*.
+
+The following screenshot shows the configuration:
+
+.Parameter Types field configuration
+image::database-parameter-types.png[Set the parameter type field to edit inline and then click the plus sign to create a new parameter]
+
+In the XML editor, the `<db:parameter-types>` configuration looks like this:
+
+[source,xml,linenums]
+----
+<db:bulk-insert queryTimeout="0" queryTimeoutUnit="SECONDS">
+     <db:sql>INSERT INTO PLANET(POSITION, NAME) VALUES (:position, :name)</db:sql>
+     <db:parameter-types>
+         <db:parameter-type key="name" type="VARCHAR" />
+         <db:parameter-type key="position" type="INTEGER" />
+     </db:parameter-types>
+ </db:bulk-insert>
+----
+
+== Configure the Column Types Field in Studio
+
+There are exclusive and common parameters for each connection provider, such as Derby and Oracle. A child element of the connection provider element defines custom data types you can use when connected to a particular provider. The following example shows how to configure the Column Type field in the Derby global configuration connection element:
```

Comment:
GLOBALLY avoid starting a sentence with "There is" or "There are"; see style ref:
```suggestion
Each connection provider uses exclusive and common parameters, such as Derby and Oracle. A child element of the connection provider element defines custom data types you can use when connected to a particular provider. The following example shows how to configure the Column Type field in the Derby global configuration connection element:
```
Are the first and second sentences related or just two separate facts about providers and their parameters and data types? I was looking for a connection between the first and second sentence.